zerver/tests/webhooks/test_bitbucket.py

```diff
@@ -119,23 +119,23 @@ def test_bitbucket2_on_pull_request_merged_event(self):
 
     def test_bitbucket2_on_pull_request_comment_created_event(self):
         # type: () -> None
-        expected_message = u"User Tomasz(login: kolaszek) created [comment](https://bitbucket.org/kolaszek/repository-name/pull-requests/3/_/diff#comment-20576503 in [\"new commit\" pull request](https://bitbucket.org/kolaszek/repository-name/pull-requests/3)"
+        expected_message = u"kolaszek [commented](https://bitbucket.org/kolaszek/repository-name/pull-requests/3/_/diff#comment-20576503) [PR](https://bitbucket.org/kolaszek/repository-name/pull-requests/3)\n\n~~~ quote\nComment1\n~~~"
```

Comment:
I think if we're mentioning the PR/Issue in the body, we should include the number in there; it just reads as clearer.

And yeah, we can do it as a separate commit.